demo/README.md

```diff
@@ -173,6 +173,19 @@ python mmediting_inference_demo.py \
         --result-out-dir ../resources/output/video_restoration/demo_video_restoration_edvr_res.mp4
 ```
 
+#### 2.2.9 EG3D
+
+```shell
+python demo/mmediting_inference_demo.py \
+    --model-name eg3d \
+    --model-config configs/eg3d/eg3d_cvt-official-rgb_shapenet-128x128.py \
```

Comment:
remove some specifications of parameters so that users can quick run this command without thinking the meaning.